src/dxgi/dxgi_factory.cpp

```diff
@@ -3,6 +3,266 @@
 
 namespace dxvk {
 
+  class DxgiSwapChainDispatcher : public ComObject<IDXGISwapChain4> {
```

Comment:
This should probably not be a `ComObject`, instead, just forward AddRef/Release to the underlying swap chain.